api.go

```diff
@@ -6,13 +6,32 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"net/url"
-	"strconv"
 
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
+	"github.com/gorilla/mux"
 )
 
+// NewAPIRouter handles any urls that begin with '/api/'
+func (es *ExploreServer) NewAPIRouter() {
+	r := es.router.PathPrefix("/api").Subrouter()
+
+	r.HandleFunc("/block/hash/{hash}", es.getBlock).
+		Methods("GET")
+	r.HandleFunc("/block/height/{height}", es.getBlockByHeight).
+		Methods("GET")
+	r.HandleFunc("/hosts/", es.getHosts).
+		Methods("GET")
+	r.HandleFunc("/status/", es.getStatus).
+		Methods("GET")
+}
+
+func writeJson(w http.ResponseWriter, obj interface{}) {
```

Comment:
less concerning if you're planning on throwing all of this go code away, which I'm guessing is what's going to happen in the long term.